webapp/src/components/history.tsx

```diff
@@ -96,7 +94,7 @@ export function History() {
               <Form.Select field="backendUrl" label='RoutedTo' style={{ width: 200 }} showClear placeholder={Locale.History.RoutedToTip}>
                 {backendData?.map(b => (
                   <Form.Select.Option key={b.externalUrl} value={b.externalUrl}>
-                    <Tag color={'blue'} style={{ marginRight: '5px' }}>{backendMapping[b.externalUrl]}</Tag>
+                    <Tag color={'blue'} style={{ marginRight: '5px' }}>{b.name}</Tag>
```

Comment:
Do we still use `backendMapping` anywhere if you change it to `b.name`? Or we can completely remove it